stampit.js

```diff
@@ -45,7 +45,7 @@ function extractFunctions(arg) {
 
 function addMethods(fixed, methods) {
   var args = [fixed.methods].concat(methods);
-  mixer.mixinFunctions.apply(null, args);
+  mixer.mixin.apply(null, args);
```

Comment:
We can't be sure. And it's good! See my comment above. TL;DR - advanced examples require this change.

In fact, I'd add `proto` alias to the `methods` method (maybe someday).